go.mod

```diff
@@ -0,0 +1,61 @@
+module github.com/lopezator/burner.kiwi
```

Comment:
I think this needs changing to point to `haydenwoodhead/burner.kiwi`

Reply:
Oops, sorry about this!